common/compositor/va/varenderer.cpp

```diff
@@ -295,6 +296,11 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   pipe_param.output_region = &output_region;
   pipe_param.output_color_standard = VAProcColorStandardBT601;
 
+  if (state.layer_->IsProtected()) {
+    /*Turn on protected flag*/
+    pipe_param.input_surface_flag = 1;
```

Comment:
Travis if failing with this error:  
```
compositor/va/varenderer.cpp: In member function virtual bool hwcomposer::VARenderer::Draw(const hwcomposer:   :MediaState&, hwcomposer::NativeSurface*):
compositor/va/varenderer.cpp:301:16: error: VAProcPipelineParameterBuffer has no member named input_su   rface_flag
 pipe_param.input_surface_flag = 1;
                 ^
make[3]: *** [compositor/va/varenderer.lo] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

Reply:
I have compiled this in OneA. But it failed on Linux

Reply:
Updated to add MACRO to control this

Reply:
Will merge this as Tong Liang said he has verified this patch can work